### PR TITLE
Fix broken Address Reuse links

### PR DESCRIPTION
**Pull request by dennisreimann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,15 +126,15 @@ And the index of the video of a playlist (e.g. starts the third video):
 
 There are three types of text highlights that can be used to display different colored boxes.
 
-A green box displaying a [friendly tip](https://docs.wasabiwallet.io/using-wasabi/AddressReuse.html):
+A green box displaying a [friendly tip](https://docs.wasabiwallet.io/why-wasabi/AddressReuse.html):
```
Comment (yahiheb) [inline README.md]:
```suggestion
A green box displaying a [friendly tip](https://docs.wasabiwallet.io/using-wasabi/PasswordBestPractices.html):
```

File: README.md [hunk 2]
```diff
@@ -126,15 +126,15 @@ And the index of the video of a playlist (e.g. starts the third video):
...
 
 ```
 :::tip
 foo
 :::
 ```
 
-A yellow box with a [cautious warning](https://docs.wasabiwallet.io/using-wasabi/AddressReuse.html):
+A yellow box with a [cautious warning](https://docs.wasabiwallet.io/why-wasabi/AddressReuse.html):
```
Comment (yahiheb) [inline README.md]:
```suggestion
A yellow box with a [cautious warning](https://docs.wasabiwallet.io/using-wasabi/WalletRecovery.html#backup-wallet-file-and-password):
```

**Comment by MaxHillebrand** [comment]
@yahiheb I'm unsure if your suggestions are resolved?
Can I merge?

**Comment by dennisreimann** [comment]
No, they aren‘t. Something went wrong here and the changes I accepted did not get committed. Will have to re-add those manually before merge.

**Comment by dennisreimann** [comment]
Now I'm confused too: The linsk as they are work. What am I missing?

**Comment by yahiheb** [comment]
> Now I'm confused too: The linsk as they are work. What am I missing?

The links you suggested work, but they don't serve the purpose for which they were add in the first place which is to show the types of the text highlights.
The first link should contain a tip (green), and the second one should contain a warning (yellow).

**Comment by dennisreimann** [comment]
@yahiheb Thanks for clarifying that – now your changes are incorporated into this PR! 🙏

**Pull request merged**